src/btree.c

```diff
@@ -3108,29 +3161,31 @@ btree_print_index_node(platform_log_handle *log_handle,
       log_handle, "Array of %d index entries:\n", btree_num_entries(hdr));
    for (uint64 i = 0; i < btree_num_entries(hdr); i++) {
       index_entry *entry = btree_get_index_entry(cfg, hdr, i);
-      platform_log(log_handle,
-                   "[%2lu]: key=%s\n"
-                   "   child_addr=%lu\n"
-                   "   (num_kvs=%u\n"
-                   "    key_bytes=%u\n"
-                   "    message_bytes=%u)\n",
-                   i,
-                   key_string(dcfg, index_entry_key(entry)),
-                   entry->pivot_data.child_addr,
-                   entry->pivot_data.stats.num_kvs,
-                   entry->pivot_data.stats.key_bytes,
-                   entry->pivot_data.stats.message_bytes);
+      btree_print_index_entry(log_handle, cfg, entry, i);
    }
    platform_log(log_handle, "\n");
 }
 
+static void
+btree_print_leaf_entry(platform_log_handle *log_handle,
+                       btree_config        *cfg,
+                       leaf_entry          *entry,
+                       uint64               entry_num)
+{
+   data_config *dcfg = cfg->data_cfg;
+   platform_log(log_handle,
+                "[%2lu]: %s -- %s\n",
+                entry_num,
+                key_string(dcfg, leaf_entry_key(entry)),
+                message_string(dcfg, leaf_entry_message(entry)));
+}
```

Comment:
Ignore this refactoring of BTree print code into sub-functions. That will be separately integrated to `/main` before this change-set under PR #521 .